<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vSongEdit</class>
 <widget class="QDialog" name="vSongEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Song Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icons.qrc">
    <normaloff>:/icons/appicon</normaloff>:/icons/appicon</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <item>
    <layout class="QHBoxLayout" name="CentralWidget">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="MainPanel">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="CmbSongbooks"/>
         </item>
         <item>
          <widget class="QToolBar" name="ToolbarEditor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="windowTitle">
            <string notr="true">toolBar1</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <addaction name="actionSave"/>
           <addaction name="actionDelete"/>
           <addaction name="actionClear"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLayout_2">
         <item>
          <widget class="QLineEdit" name="TxtNumber">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Song Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TxtTitle">
           <property name="placeholderText">
            <string>Title of the Song</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="TxtContent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Content of the Song</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLayout_3">
         <item>
          <widget class="QLabel" name="LblOptional">
           <property name="text">
            <string>Optional</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TxtAuthor">
           <property name="placeholderText">
            <string>Song Author (Optional)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TxtKey">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="placeholderText">
            <string>Song Key (Optional)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string> Content</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TxtAlias">
           <property name="placeholderText">
            <string>Alias of the Song (i.e in English if not an English Song)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
   <property name="iconText">
    <string>Save Changes</string>
   </property>
   <property name="toolTip">
    <string>Save Changes</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Song</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/cancel</normaloff>:/icons/cancel</iconset>
   </property>
   <property name="text">
    <string>Clear Text</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
